src/DynamicNLPModels.jl

```diff
@@ -6,7 +6,7 @@ import LinearAlgebra
 import SparseArrays
 import SparseArrays: SparseMatrixCSC
 
-export LQDynamicData, SparseLQDynamicModel, DenseLQDynamicModel, get_u, get_s
+export LQDynamicData, SparseLQDynamicModel, DenseLQDynamicModel, get_u, get_s, Block_Matrices
```

Comment:
The type name should be `BlockMatrices`. Or I'd call it `DenseLQDynamicBlocks`. But is it necessary for this to be exported?

Reply:
I originally exported these because I was rebuilding a `DenseLQDynamicModel` that contained `CuArrays`. To do this, I need the `BlockMatrices` structure so that I could pass it to `DenseLQDynamicModel`. But I agree that it should be renamed regardless.

Reply:
You can do it without exporting. To me, `BlockMatrices` should be treated as an internal data structure